### PR TITLE
[improve][kinesis-sink] Add unit test for Kinesis sink JsonConverter

### PR DESCRIPTION
**Pull request by cbornet**
### Motivation
Improves test coverage

### Modifications
Added a test

### Verifying this change

- [x] Make sure that the change passes the CI checks.

This change added tests and can be verified as follows:

Run JsonConverterTests

### Does this pull request potentially affect one of the following parts:

no

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API: (yes / no)
  - The schema: (yes / no / don't know)
  - The default values of configurations: (yes / no)
  - The wire protocol: (yes / no)
  - The rest endpoints: (yes / no)
  - The admin cli options: (yes / no)
  - Anything that affects deployment: (yes / no / don't know)

### Documentation

Check the box below or label this PR directly.

Need to update docs? 

- [ ] `doc-required` 
(Your PR needs to update docs and you will update later)
  
- [x] `doc-not-needed` 
fix
  
- [ ] `doc` 
(Your PR contains doc changes)

- [ ] `doc-complete`
(Docs have been already added)

### CODE REVIEW

**Comment by cbornet** [comment]
> @cbornet isn't it the same of https://github.com/apache/pulsar/pull/15432 ?

Yes it is. Sorry I hadn't seen it.
I'll modify it to only keep the unit test

File: pulsar-io/kinesis/src/test/java/org/apache/pulsar/io/kinesis/json/JsonConverterTests.java [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class JsonConverterTests {
+
+    @Test
+    public void testAvroToJson() throws IOException {
+        Schema avroArraySchema = SchemaBuilder.array().items(SchemaBuilder.builder().stringType());
+        Schema schema = SchemaBuilder.record("record").fields()
```
Comment (shibd) [inline pulsar-io/kinesis/src/test/java/org/apache/pulsar/io/kinesis/json/JsonConverterTests.java:53]: Suggestion adds Logical type test. Remind, `JsonNode` will default to the scientific and technical method when acquiring big data types. refer to: #15687

Comment (cbornet) [reply pulsar-io/kinesis/src/test/java/org/apache/pulsar/io/kinesis/json/JsonConverterTests.java:53]: This PR is mainly to align with [Elastic Search JsonConverter tests](https://github.com/apache/pulsar/blob/master/pulsar-io/elastic-search/src/test/java/org/apache/pulsar/io/elasticsearch/JsonConverterTests.java). Maybe we could improve with Logical types in a follow-up PR for both ?

**Pull request merged**